src/XenCert/XenCertLog.py

```diff
@@ -0,0 +1,58 @@
+import sys
+sys.path.insert(0, "/opt/xensource/sm")
+from util import SMlog
+import commands
+import os
```

Comment:
I suggest put python own modules in front, our private module in tail.

Reply:
Done

Reply:
@DeliZhangX  Before you merged my changes,  I will run testing again.

Reply:
Thanks @MarkSymsCtx and @DeliZhangX for your review.